docs/src/main/tut/docs/combinators.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+title: Combinators
+---
+
+## {{page.title}}
+
+The combinators defined in `ConfigReader` provide an easy way to create new `ConfigReader` instances by transforming
+existing ones. They are the simplest solution for supporting new simple types and for slightly modifying existing
+implementations, since the amount of boilerplate required is very small. This section contains some examples of
+combinators and shows how to work with them in PureConfig.
+
+The simplest combinator is `map`, which simply transforms the result of an existing reader:
+
+```tut:silent
+import com.typesafe.config.ConfigFactory
+import pureconfig._
+
+case class Conf(bytes: Vector[Byte])
+
+// reads an array of bytes from a string
+implicit val byteVectorReader = ConfigReader[String].map(_.getBytes.toVector)
```

Comment:
Nitpick: I'd rather see a type annotation, `ConfigReader[Vector[Byte]]`, to reduce ambiguity. Feel free to ignore.